CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ Enhancements:
 
 Deprecations:
  - Removed support for the .NET Framework < 4.5 and .NET Standard 1.x. (@stakx, #495, #496)
+ - Removed support for Code Access Security (CAS). (@stakx, #502)
+ - The following public members have been removed:
+    - `Castle.Core.Internal.PermissionUtil.IsGranted` (method)
```

Comment:
Note that Windsor is currently using this method, so we will have some work to do when upgrading it to Castle.Core v5.

Reply:
I'm hitting this now - not sure of the best way forward. Could bring the method into Windsor instead, or remove CAS from Windsor too, although I've not used it enough to be entirely sure of some of the consequences.

Reply:
@Jevonius, I would suggest that you open an issue over at the Windsor repo (if you haven't already) so there is a dedicated place to discuss removal of CAS support. Until a final decision has been reached there, you could perhaps start by trying to keep the code changes minimal, i.e. simply reintroduce the required method over in Windsor (but keep it `private` / `internal` so it doesn't end up as an additional public API).

Reply:
Good suggestion, thanks. There's various other CAS-related attributes dotted around, so warrants a wider discussion. Will replicate these classes for now, but internal/private as suggested.